R/calc_aggregate_counts.R

```diff
@@ -29,6 +29,28 @@
 #' calc_aggregate_counts(state = TRUE, all_dates = TRUE)
 #' @export
 
+sub_vax_data <- function(df, ad){
```

Comment:
At first, I didn't know what `ad` was. Since we're using `all_dates` in other places, I'd prefer this function argument to be changed to `all_dates` to maintain consistency.

Reply:
Separately... I'm going to ask you to the same 2 things I asked Hope:
1. I'd like this new function to come after calc_aggregate_counts. My logic here is that you should be able to read through the main function first, then dig into supplementary functions as needed/if you care.
 2. I'd like to use the [naming convention that Google uses](https://google.github.io/styleguide/Rguide.html) for a private/helper function, which is starting the name of the function with a `.`. I like the way this delineates a core function from a helper function.